op_builder/evoformer_attn.py

```diff
@@ -41,29 +41,34 @@ def nvcc_args(self):
         args.append(f"-DGPU_ARCH={major}{minor}")
         return args
 
-    def is_compatible(self, verbose=True):
+    def is_compatible(self, verbose=False):
```

Comment:
Hi @terry-for-github - thanks for your PR, could you resolve the merge conflicts and then we can get this PR merged?

Reply:
Sure! The conflicts are resolved now.